pages/learn/develop/local-mode.md

```diff
@@ -14,25 +14,25 @@ To use local mode on a device:
 - The device must be running {{ $names.os.lower }} v2.29.0 or higher.
 - The device must be running a [development][development] variant of the OS.
 - You must have the [{{ $names.company.lower }} CLI][cli] installed on your development machine.
-- Local mode must be enabled through the {{ $names.cloud.lower }} dashboard. You can enable it from either the *Actions* menu of the device dashboard or click to expand the arrow located on the device dashboard and select *Enable Local Mode*.
+- Local mode must be enabled through the {{ $names.cloud.lower }} dashboard. You can enable it from the device _Settings_ tab.
 
-![Enable local mode](/img/local-mode/enable-local-mode.png)
+![Enable local mode](/img/local-mode/enable-local-mode-settings-tab.png)
```

Comment:
```suggestion
![Enable local mode](/img/local-mode/enable-local-mode.png)
```

Highly recommend not changing image file names if you can. This file name change would break the Getting Started guides which also reference this change https://github.com/balena-io/docs/blob/3c455e72133ac47fb038f53011ef3d0c71d4af44/pages/learn/develop/local-mode.md#L10-L19